packages/demo-wallet-client/webpack.dev.js

```diff
@@ -8,7 +8,7 @@ module.exports = merge(common, {
   devServer: {
     static: path.join(__dirname, "build"),
     historyApiFallback: true,
-    port: 3000,
+    port: 4000,
```

Comment:
`3000` port was clashing with the Anchor Platform, so I changed it to something else to avoid that.